js/controller.js

```diff
@@ -355,7 +376,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     onContentTreeFetched: function (event, contentTree) {
       this.state.contentTree = contentTree;
       this.state.playerState = CONSTANTS.STATE.START;
-      this.renderSkin({"contentTree": contentTree});
+      var duration = Utils.ensureNumber(contentTree.duration, 0) / 1000;
+      this.skin.updatePlayhead(null, duration);
```

Comment:
why the fist argument on updatePlayhead is set to null?

Reply:
I modified `updatePlayhead` so that it uses the current values if they are not passed. The idea here is that we don't know the playhead at this point, so we only update the value that we have.